static/app/views/performance/transactionSummary/transactionOverview/charts.tsx

```diff
@@ -200,19 +203,28 @@ function TransactionSummaryCharts({
           />
         )}
         {display === DisplayModes.TREND && (
-          <TrendChart
-            trendFunction={trendFunction}
-            trendParameter={trendColumn}
-            organization={organization}
-            query={eventView.query}
-            queryExtra={releaseQueryExtra}
-            project={eventView.project}
-            environment={eventView.environment}
-            start={eventView.start}
-            end={eventView.end}
-            statsPeriod={eventView.statsPeriod}
-            withoutZerofill={withoutZerofill}
-          />
+          <Feature features={['organizations:performance-new-trends']}>
+            {({hasFeature}) => {
+              return (
+                <TrendChart
+                  eventView={eventView}
+                  trendFunction={trendFunction}
+                  trendParameter={trendColumn}
+                  organization={organization}
+                  query={eventView.query}
+                  queryExtra={releaseQueryExtra}
+                  project={eventView.project}
+                  environment={eventView.environment}
+                  start={eventView.start}
+                  end={eventView.end}
+                  statsPeriod={eventView.statsPeriod}
+                  withoutZerofill={withoutZerofill}
+                  projects={project ? [project] : []}
+                  withBreakpoint={hasFeature}
```

Comment:
Since there's no conditional rendering you don't need `<Feature>`. You can srt `withBreakpoint` value by reading `organization.features` directly like:
```
withBreakpoint={organization.features.has('performance-new-trends')`